src/main/resources/swagger.api/mod-entities-links.yaml

```diff
@@ -111,6 +135,43 @@ components:
         - bibRecordTag
         - bibRecordSubfields
 
+    uuidCollection:
+      type: object
+      title: Collection of UUIDs
+      description: Collection of UUIDs
+      properties:
+        ids:
+          type: array
+          items:
+            $ref: '#/components/schemas/uuid'
+      required:
+        - ids
+
+    linkCountMapDtoCollection:
+      type: object
+      title: Collection of links count map for authorities
+      description: Collection of links count map for authorities
+      properties:
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/linkCountMapDto'
+      required:
+        - links
+
+    linkCountMapDto:
```

Comment:
I think we shouldn't use 'map' word. `linksCountDtoCollection` and `linksCountDto` is enough

Reply:
done